public/view_users.php

```diff
@@ -12,11 +12,31 @@
 } catch (PDOException $error) {
     echo $sql . "<br>" . $error->getMessage();
 }
+$user_role = get_current_role();
+
+if (isset($_POST['delete_user'])) {
+    $user_to_delete = User::find_by(array('id' => $_POST['user_id']));
+    $delete_sucess = false;
+    
+    if (!$user_to_delete->is_admin){
+        $user_to_delete->delete();
+        
+    }else {
+        echo "Admin cannot be deleted";
+    }
+
+    if($delete_sucess) {
+        echo "User successfully deleted";
+        header("refresh: 1");
+    }
```

Comment:
hmm this never shows, you never assign it to true

Reply:
done